apps/worker/src/app/workflow/usecases/subscriber-job-bound/subscriber-job-bound.usecase.ts

```diff
@@ -167,6 +157,25 @@ export class SubscriberJobBound {
       })
     );
   }
+  private buildCommand(
```

Comment:
Nit-picky comment: This extra method is not necessary, creating the command directly is more readable for me.